pkg/relayer/config/relayminer_configs_reader.go

```diff
@@ -21,15 +25,28 @@ func ParseRelayMinerConfigs(configContent []byte) (*RelayMinerConfig, error) {
 		return nil, ErrRelayMinerConfigUnmarshalYAML.Wrap(err.Error())
 	}
 
+	// Fallback to DefaultRequestTimeoutSeconds const if none is specified in the config
+	// This is for the backwards compatibility with the previous versions of the config file
+	if yamlRelayMinerConfig.DefaultRequestTimeoutSeconds == 0 {
+		yamlRelayMinerConfig.DefaultRequestTimeoutSeconds = DefaultRequestTimeoutSeconds
+	}
+
 	// Global section
 	relayMinerConfig.DefaultSigningKeyNames = yamlRelayMinerConfig.DefaultSigningKeyNames
+	relayMinerConfig.DefaultRequestTimeoutSeconds = yamlRelayMinerConfig.DefaultRequestTimeoutSeconds
 
 	// SmtStorePath is required
 	if len(yamlRelayMinerConfig.SmtStorePath) == 0 {
 		return nil, ErrRelayMinerConfigInvalidSmtStorePath
 	}
 	relayMinerConfig.SmtStorePath = yamlRelayMinerConfig.SmtStorePath
 
+	// EnableOverServicing is a flag that indicates whether the relay miner server
+	// should enable over-servicing for the relays it serves.
+	// Over-servicing allows the relay miner to serve more relays than the
+	// amount of stake the Application has.
```

Comment:
```suggestion
	//
	// Over-servicing allows the offchain relay miner to serve more relays than the
	// amount of stake the onchain Application can pay the corresponding onchain 
	// Supplier at the end of the session
	//
	// This can enable high quality of service for the network and earn quality points with Gateways.
```